bob-backend/src/core.rs

```diff
@@ -442,10 +454,12 @@ impl Backend {
     }
 
     pub async fn delete(&self, key: BobKey) -> Result<u64, Error> {
```

Comment:
This probably should be updated after https://github.com/qoollo/bob/pull/621 will be merged